src/App.js

```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+// eslint-disable-next-line
+import Calculator from './components/Calculator.js';
 
```

Comment:
- [ ] [  Optional  ]You did a great job 😃 at the required changes and the project is working now, but I see that you disable the linter at this part and it is not correct and best practice, so it will be better if you remove the disabled comment to let the linter work to discover the errors at your base code and enhance the code of the project 👌😃